### PR TITLE
Remove duplication in docs

### PR DESCRIPTION
**Pull request by jpm-canonical**
## Summary

Remove installation and common usage instruction which are already included in the main documentation [here](https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/otbr-on-ubuntu/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,44 +12,37 @@ OpenThread Border Router source code: https://github.com/openthread/ot-br-posix
...
->
+```bash
+sudo snap install --dangerous *.snap
+```
 
-### Grant access to resources
+### Connect interfaces
+
+When installing this snap from the store, some interfaces are automatically connected.
+If you install the snap from a local file, like the one you built yourself, you need to connect these interfaces manually:
```
Comment (farshidtz) [inline README.md:44]:
```suggestion
When installing this snap from the store, some interfaces are automatically connected.
If you install the locally built snap, you need to connect these interfaces manually:
```

File: README.md [hunk 2]
```diff
@@ -64,30 +58,27 @@ sudo snap connect openthread-border-router:bluetooth-control
...
 ```bash
-sudo snap start openthread-border-router
+sudo snap install avahi bluez
+sudo snap connect openthread-border-router:avahi-control avahi:avahi-control
+sudo snap connect openthread-border-router:bluez bluez:service
 ```
-Add `--enable` flag to make the service start on boot as well.
 
-To query and follow the logs: `snap logs -n 10 -f openthread-border-router`
+## Advanced Usage
```
Comment (farshidtz) [inline README.md:71]: I suggest removing this whole section and migrating the following to [this section](https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/otbr-on-ubuntu/#start-otbr) of the documentation:
```
By default, the services are disabled and not started.
```
```
To start and enable via a [Gadget snap](https://snapcraft.io/docs/the-gadget-snap), set `autostart` to `true`.
```

**Review by farshidtz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jpm-canonical) [reply README.md:71]: https://github.com/canonical/matter-docs/pull/40

**Review by farshidtz — APPROVED** [review]
LGTM

**Pull request merged**